cogs/csgo.py

```diff
@@ -114,6 +132,8 @@ async def pug(self, ctx: commands.Context, *args):
             embed = self.player_veto_embed(message_text=message_text, players_text='Random Teams', team1=team1,
```

Comment:
```suggestion
            embed = CSGO.player_veto_embed(message_text=message_text, players_text='Random Teams', team1=team1,
```
The function `player_veto_embed` is now a static function, same on line 199 and 258

Reply:
Fixed Static method thing